apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,31 @@
+import { IDetailQueryResult, ITranscribedAudioViewModel } from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import './transcribed-audio-detail.full-view.presenter.css';
+
+export const TranscribedAudioDetailFullViewPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const runTime = Math.round(transcribedAudioItem.length / 1000);
+
+    return (
+        <div
+            className="detail-full-view-container"
+            title="View Connected Photograph"
+            data-testid={transcribedAudioItem.id}
+        >
+            <div className="detail-full-view-media-container">
+                <MediaPlayer audioUrl={transcribedAudioItem.audioURL} />
+            </div>
+            <div className="detail-full-view-meta-container">
+                <strong>Transcribed Audio ID:</strong> {transcribedAudioItem.id}
```

Comment:
This is ok for now. It's kind of hard to read, though. If this is where the dust settles, we should have a helper that reduces over and wraps in the `<strong> ... </strong><br/>` blocks.